### PR TITLE
[Databricks] DBFS Double Encryption Support

### PR DESCRIPTION
**Pull request by isra-fel**
## Description

Generation PR: #12517

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Databricks/Az.Databricks.format.ps1xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
       </ViewSelectedBy>
       <TableControl>
         <TableHeaders>
+          <TableColumnHeader>
+            <Label>PeeringName</Label>
```
Comment (isra-fel) [inline src/Databricks/Az.Databricks.format.ps1xml:12]: Vnet peerings commands are removed from generating, but the models are still generated. That's why there are so many files in this PR

File: src/Databricks/Az.Databricks.psd1 [hunk 2]
```diff
@@ -108,7 +108,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Added support for customer-managed keys.'
+        # ReleaseNotes = ''
```
Comment (isra-fel) [inline src/Databricks/Az.Databricks.psd1:111]: Is this OK?

File: src/Databricks/test/Az.Databricks-TestResults.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (isra-fel) [inline src/Databricks/test/Az.Databricks-TestResults.xml:1]: Should this file be excluded?

File: src/Databricks/help/Az.Databricks.md [hunk 4]
```diff
@@ -10,6 +10,10 @@ Locale: en-US
 ## Description
 Microsoft Azure PowerShell: Databricks cmdlets
 
+While the Az.Databricks PowerShell module is in preview, you must install it separately from the Az PowerShell module using the following command: `Install-Module -Name Az.Databricks`. Once the Az.Databricks PowerShell module is generally available, it becomes part of future Az PowerShell module releases and available natively from within Azure Cloud Shell.
```
Comment (isra-fel) [inline src/Databricks/help/Az.Databricks.md:13]: This was accidently deleted in last PR.  I added it back

File: src/Databricks/Az.Databricks.psd1 [hunk 5]
```diff
@@ -12,7 +12,7 @@
 RootModule = './Az.Databricks.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.1.1'
+ModuleVersion = '0.2.0'
```
Comment (dingmeng-xue) [inline src/Databricks/Az.Databricks.psd1:15]: version should not be changed. Please check with Yunchi. He knows the issue I met.

Comment (isra-fel) [reply src/Databricks/Az.Databricks.psd1]: Talked with Yunchi and I rolled it back. Thanks

File: src/Databricks/test/Az.Databricks-TestResults.xml (hunk 3 above)
Comment (dingmeng-xue) [reply src/Databricks/test/Az.Databricks-TestResults.xml:1]: yes.

Comment (dingmeng-xue) [reply src/Databricks/test/Az.Databricks-TestResults.xml:1]: Can it be removed?

Comment (isra-fel) [reply src/Databricks/test/Az.Databricks-TestResults.xml]: Removed

**Pull request merged**